### DISCUSSION TITLE
How do I make the start of YAxis and XAxis overflowing like in this image?

### DISCUSSION THREAD

**Discussion by handhikadj** [recharts/recharts#3968] [Q&A]
<img width="157" alt="Screenshot 2023-11-16 at 1 05 45 PM" src="https://github.com/recharts/recharts/assets/22522638/3cda56ee-301d-4414-adc4-5640fda5cae4">

can't find on the examples. please help

**Comment by ckifer** [comment]
Can you give more details please? What have you tried, a codesandbox with your current code, etc?

As far as I know this isn't possible without custom SVGs

This gets you the color and the stroke width
https://codesandbox.io/s/simple-line-chart-forked-x5wvqj?file=/src/App.tsx

Otherwise you can make your own axis component and change the x/y + width

**Reply by handhikadj** [reply]
I can't show any code because I want to ask the possibility of doing this, like the workaround of it.
never thought that it would be that difficult to make it overflown.
thanks for commenting anyway

**Reply by ckifer** [reply]
Feel free to make a feature request issue if this is something you want made easier. 

Custom SVG will work, but will just take a bit more time to figure out (I can take a stab at it when I have time)
